proposals/0000-template.rst

```diff
@@ -52,6 +52,9 @@ may include,
 * how the proposed change interacts with existing language or compiler
   features, in case that is otherwise ambiguous
 
+The ideal specification is described as a modification of the
+`Haskell 2010 report <https://www.haskell.org/definition/haskell2010.pdf>`_.
+
 Note, however, that this section need not describe details of the
 implementation of the feature or examples. The proposal is merely supposed to
 give a conceptual specification of the new feature and its behavior.
```

Comment:
replace "conceptual" with "precise".   Also "need not" should be "should not".  And we should distinguish examples.  How about this?

"Note, however, that this section should focus on a precise *specification*; it need not (and should not) devote space to *implementation* details -- there is a separate section for that.

"The specification can, and almost always should, be illustrated with *examples* that illustrate corner cases.  But it is not sufficient to give a couple of examples and regard that as the specification!  The examples should illustrate and elucidate a clearly-articulated specification that covers the general case."